bamboo/Dockerfile

```diff
@@ -5,6 +5,19 @@ RUN apt update && npm config set unsafe-perm true &&\
 RUN apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
 libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
 xz-utils tk-dev libffi-dev liblzma-dev python3-pip
+
+# install python
```

Comment:
Currently testing. We have an issue in CI where it's using the container's 3.10 version and that may be causing problems.